tests/unit/test.errors.js

```diff
@@ -34,9 +34,11 @@ describe('errors', () => {
 
     class ErrorWithCode extends Error {
       code: string;
+      errno: string;
```

Comment:
instead of editing this class, I think you need to create a new class named something like `ErrorWithErrno` and add `errno` to that one only. Otherwise, the existing tests for error codes will not be as effective.